kolibri/plugins/device/assets/src/views/PostSetupModalGroup.vue

```diff
@@ -100,6 +103,9 @@
         }
         return null;
       },
+      classesLoaded() {
+        return Array.isArray(this.classes);
```

Comment:
This will return `true` for an empty array, which might cause some unexpected behavior, or that might be the intention that you have.

Is your goal for this to return true only when there are classes? Or is it just to ensure that classes is no longer undefined, and if there are _no_ classes (an empty array), that's okay?